modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/components/Steps/UploadDocuments/UploadFiles.js

```diff
@@ -2,17 +2,27 @@ import ClayIcon from '@clayui/icon';
 import React, {useState} from 'react';
 
 import {InfoBadge} from '~/common/components/fragments/Badges/Info';
+import {RemoveDocument} from '../../../services/DocumentsAndMedia';
+
 import DropArea from '../../drop-area';
 
 import ViewFiles from './ViewFiles';
 
 const UploadFiles = ({dropAreaProps, files, setFiles, title}) => {
 	const [showBadgeInfo, setShowBadgeInfo] = useState(false);
 
-	const onRemoveFile = ({id}) => {
-		const newList = files.filter((file) => file.id !== id);
+	const onRemoveFile = (_file) => {
+		try {
+			if (typeof DeleteDocument === 'function' && _file.documentId) {
```

Comment:
RemoveDocument function will always be a function, to you can remove this typeof validation.

Reply:
From _file you just get the documentId and id. 

You can destructure this way

 (_file) -> ({documentId, id})